automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -422,6 +422,17 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     val testResult: Try[T] = Try {
       testCode(cluster)
     }
+
```

Comment:
Just thinking out loud here. If `monitorCreate` is true, then the sequence will be:
1. Create
2. Wait till Running
3. Run test code
4. Delete

When `monitorCreate` is false, the sequence will be:
1. Create
2. Run test code
3. Wait till Running or Error
4. Delete

Yeah, I think this makes sense -- I like how we tolerate Error status in the second scenario. 👍